crypto/ec/curve448/ed448.h

```diff
@@ -198,10 +203,11 @@ c448_error_t curve448_point_decode_like_eddsa_and_mul_by_ratio(
  * x (out): The ECDH private key as in RFC7748
  * ed (in): The EdDSA private key
  */
-c448_error_t c448_ed448_convert_private_key_to_x448(
-                            OSSL_LIB_CTX *ctx,
-                            uint8_t x[X448_PRIVATE_BYTES],
-                            const uint8_t ed[EDDSA_448_PRIVATE_BYTES],
-                            const char *propq);
+c448_error_t
+ossl_c448_ed448_convert_private_key_to_x448(
```

Comment:
This is quite a mouthful!  (and yeah, I know there are worse examples...)

Reply:
I suppose the c448 prefix could be just dropped here and in other similarly named functions.

Reply:
It could be abbreviated to `ossl_ed448_privkey2x448` or `ossl_ed448_privkey_to_x448`

Reply:
I prefer not to do this, as it ends up with name clashes..
 There are already ossl_ed448_sign & ossl_ed448_verify for example.